hack/build.sh

```diff
@@ -0,0 +1,89 @@
+# To push to your own registry, override the REGISTRY and NAMESPACE env vars,
+# i.e:
+#   $ REGISTRY=quay.io NAMESPACE=yourusername ./hack/build.sh
+#
+# REQUIREMENTS:
+#  * a valid login session to a container registry.
+#  * `docker`
+#  * `jq`
+#  * `yq`
+#  * `opm`
+#  * `skopeo`
+
+set -e
+
+export OPERATOR_NAME='observability-operator'
+export REGISTRY=${REGISTRY:-'quay.io'}
+export NAMESPACE=${NAMESPACE:-'rhobs'}
+export TAG=${TAG:-'0.0.23'}
+export CSV_PATH=${CSV_PATH:-'bundle/manifests/observability-operator.clusterserviceversion.yaml'}
+export ANNOTATIONS_PATH=${ANNOTATIONS_PATH:-'bundle/metadata/annotations.yaml'}
+
+function cleanup {
+	# shellcheck disable=SC2046
+	if [ -x $(command -v git >/dev/null 2>&1) ]; then
+		git checkout "${CSV_PATH}" >/dev/null 2>&1
+		git checkout "${ANNOTATIONS_PATH}" >/dev/null 2>&1
+	fi
+}
+
+trap cleanup EXIT
+
+# prints pre-formatted info output.
+function info {
+	echo "INFO $(date '+%Y-%m-%dT%H:%M:%S') $*"
+}
+
+# prints pre-formatted error output.
+function error {
+	>&2 echo "ERROR $(date '+%Y-%m-%dT%H:%M:%S') $*"
+}
```

Comment:
Could you please consider reusing the functions in https://github.com/rhobs/observability-operator/blob/main/test/lib/utils.bash 
Perhaps we should move the functions to hack/lib instead.